internal/services/contracts_events_consumer.go

```diff
@@ -698,6 +711,136 @@ func (c *ContractsEventsConsumer) aftermarketDeviceAddressReset(e *ContractEvent
 	return err
 }
 
+// vehicleNodeBurned handles the event of the same name from the registry contract.
+func (c *ContractsEventsConsumer) vehicleNodeBurned(e *ContractEventData) error {
```

Comment:
We're handling this in the `Transfer(address,address,uint256)` now. I like to stick with "basic" events if I can, they're more likely to be emitted.

Reply:
ah, snuck in with an out of date git merge!